### ISSUE TITLE
ImageShape: alt text is missing

### ISSUE THREAD

**Issue by Pareder** [tldraw/tldraw#8152]
In `ImageShape` there are 2 different images rendered: one with alt text support and the other without. Is there any reason why loaded src is rendered with empty alt https://github.com/tldraw/tldraw/blob/main/packages/tldraw/src/lib/shapes/image/ImageShapeUtil.tsx#L443?

**Comment by Pareder** [comment]
@mimecuvalo Thank you for your PR. Will it possible to fix it in v3 as well?

**Comment by mimecuvalo** [comment]
Sorry, we aren't retroactively making patches for v3 unless for security/data-loss type concerns.
